src/index.html

```diff
@@ -10,23 +10,39 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar for Airbnb</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form--big"
+      data-qa="big"
+    >
+      <label class="search-form__label search-form__label--big">
+        <input
+          class="search-form__input search-form__input__big"
```

Comment:
The class name 'search-form__input__big' should use a single underscore for BEM naming convention, like 'search-form__input--big', to match the convention used elsewhere in the file.